server/src/BGS.Api/Program.cs

```diff
@@ -1,12 +1,18 @@
 using BGS.Api.Middleware;
 using BGS.Api.ServiceInstallers.Extensions;
+using BGS.Infrastructure.Data;
+using Microsoft.EntityFrameworkCore;
 
 var builder = WebApplication.CreateBuilder(new WebApplicationOptions
 {
     WebRootPath = Path.Combine(Directory.GetCurrentDirectory(), "client_dist"),
     Args = args,
 });
 
+var connection = builder.Configuration.GetConnectionString("DefaultConnection");
+builder.Services.AddDbContext<ApplicationDbContext>(options =>
+    options.UseNpgsql(connection, b => b.MigrationsAssembly("BGS.Infrastructure")));
```

Comment:
> `b.MigrationsAssembly("BGS.Infrastructure"))`

What is this setup for?
I think it's not necessary. For create migration you can use this command:
`dotnet ef migrations add MIGRATIONNAME -c ApplicationDbContext -p ../BGS.Infrastructure/BGS.I
nfrastructure.csproj -s BGS.Api.csproj -o Migrations`

Reply:
Do you think it might be necessary to move the Migration folder to inner Data folder?
I think it's more logical

Reply:
> For create migration you can use this command:
For creating migration I use Rider's plugin ".NET EF migration facilitator". If it doesn't work I use VS terminal and simple command `add-migration + migration name` 

> Do you think it might be necessary to move the Migration folder to inner Data folder?
I thought about this, maybe it is better to move this folder inside "Data" folder. I will check how it works after all changes